terraform/examples/ces_google_gke/create_bucket/variables_gke.tf

```diff
@@ -0,0 +1,45 @@
+variable "gcp_project_name" {
+  type = string
+}
+
+variable "gcp_zone" {
+  type    = string
+  default = "europe-west3-c"
+}
+
+variable "gcp_region" {
+  type    = string
+  default = "europe-west3"
+}
+
+variable "gcp_credentials" {
+  type      = string
+  sensitive = true
+  default   = "../secrets/gcp_sa.json"
+}
+
+variable "create_bucket" {
+  type = bool
+  default = true
+}
+
+variable "bucket_name" {
+  description = "The name of the bucket"
+  type        = string
+  default     = "cloudogu-backup-bucket"
+}
+
+variable "use_bucket_encryption" {
+  type = bool
+  default = true
+}
+
+variable "key_ring_name" {
+  type    = string
+  default = "ces-key-ring"
```

Comment:
change default value to not say "ces" - cloudogu would be better. Same with key_name